### DISCUSSION TITLE
UAC prompts and app tray context menus

### DISCUSSION THREAD

**Discussion by BlueGoliath** [LizardByte/community#137] [Sunshine] [answered]
Hi,

When a UAC prompt is shown on Sunshine 0.17 the display freezes and keyboard/mouse input is ignored. Similarly when bringing up the context menu of an app in the app tray(e.g. Nvidia's control panel) keyboard and mouse input is lost.

This is with a Windows 10 host streaming to a Linux PC.

Is this a known limitation? I don't remember this ever being an issue with Nvidia's official server.

**Comment by ReenigneArcher** [comment, marked as answer]
Normally you can run Sunshine as a service to show elements within the SYSTEM/ADMIN context, such as UAC prompts.

**Reply by BlueGoliath** [reply]
Thanks for the quick reply.

Can you tell me how I run it as a service?

**Reply by ReenigneArcher** [reply]
Easiest way is to use our installer with the default options.

**Reply by BlueGoliath** [reply]
I found the issue. After installing Sunshine I was starting a new instance in admin mode via the start menu. After a reboot I noticed that Sunshine now starts at startup and the issue no longer happens. This also seems to explain why none of the NVENC settings I was changing made any difference.

Suggestion: warn if Sunshine is already running or if the service is installed and not running.

Thanks.
